pytorch_lightning/plugins/training_type/horovod.py

```diff
@@ -158,3 +158,7 @@ def gather_all_tensors(self, result: Union[torch.Tensor], group: Optional[Any] =
         gathered = hvd.allgather(result)
         gathered_result = list(gathered.split(1, dim=0))
         return gathered_result
+
+    def post_backward(self, loss: torch.Tensor, should_accumulate: bool):
+        for optimizer in self.optimizers:
+            optimizer.synchronize()
```

Comment:
This method is duplicated, it is also above in line 119

Reply:
Great catch !